Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -39,9 +39,11 @@
    "source": [
     "# Uncomment & specify proxy value if applicable\n",
     "# %env https_proxy=http://<proxy_host>:<port>\n",
+    "# Comma separated list of IP address\n",
+    "# %env no_proxy=\"<host1>:<port1>,<host2>:<port2>,...\"\" \n",
```

Comment:
there could be not only IPs, but also IPs with masks and text hostnames. And afaik, it can not contain ports contrrary to httss_proxy.

Reply:
Updated the comment and also added list in the `no_proxy`  as an example to provide better clarity. If anything else is also needed, please update me.